### DISCUSSION TITLE
Properties panel not updating when selecting a new transition after selecting a previous one

### DISCUSSION THREAD

**Discussion by Adem-Kri** [eclipse-glsp/glsp#1153] [Q&A]
Hello,
I have a graph where I'm able to drag and drop elements and have the possibility to link them using transitions. Each time I select a transition, a properties panel opens containing different information about the selected transition. 

However, I've noticed a bug where if I select a transition and then select another one without selecting an element in between, the properties panel does not update with the newly selected transition's information. Instead, it continues to display the information of the first transition that was selected.
Can you help me with this bug

Thank you in advance.

**Comment by rsoika** [comment]
Hi @Adem-Kri ,
I have implemented a PropertyPanel in my own [project](https://www.open-bpmn.org/glsp-client/BPMN_PROPERTIES.html) as an UI-Extension. I wonder how you have implemented your property panel. A property panel is not a standard feature in GLSP so I wonder what you have done in your project? Maybe you can share a screenshot? Also updating a property panel requires some custom selectionListener.

**Comment by tortmayr** [comment]
Hi @Adem-Kri,
as @rsoika already mentioned it is kind of hard to help you with this issue without additional implementation details. 
How do you forward the diagram selection to you custom property panel?
